docs/boot-server/ipxe.cfg.j2

```diff
@@ -0,0 +1,44 @@
+#!ipxe
+
+cpair --foreground 6 --background 9 1
+cpair --foreground 7 --background 9 2
+cpair --foreground 6 --background 9 3
+
+console --picture http://{{ inventory_hostname }}/tateru/ipxe.png
+
+set ipparam BOOTIF=${netX/mac} ip=${ip}::${gateway}:${netmask}
+set kp initrd=amd-ucode.img initrd=intel-ucode.img initrd=initramfs-linux.img
+set kp ${kp} loglevel=3 cow_spacesize=768M archisobasedir=arch
+set kp ${kp} archiso_http_srv=http://{{ inventory_hostname }}/tateru/ verify=n
```

Comment:
We should probably not use inventory_hostname here but instead ansible_fqdn or get the actual IP from gather facts.